sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/jms/ServiceBusJmsConnectionFactoryFactory.java

```diff
@@ -26,27 +27,31 @@
 
     }
 
-    <T extends ServiceBusJmsConnectionFactory> T createConnectionFactory(Class<T> factoryClass) {
-        try {
-            return doCreateConnectionFactory(factoryClass);
-        } catch (Exception ex) {
-            throw new IllegalStateException("Unable to create ActiveMQConnectionFactory", ex);
-        }
+    <T extends ServiceBusJmsConnectionFactory> T createConnectionFactory(Class<T> factoryClass)throws IllegalStateException {
```

Comment:
why adding the `throws IllegalStateException `

Reply:
How about still try catch the `createConnectionFactoryInstance `

Reply:
remote the throws state and let the exception throws by default.